docs/io.cozy.accounts.md

```diff
@@ -25,18 +26,11 @@ Accounts can be managed in [Cozy-Home](http://github.com/cozy/cozy-home/) (via [
 
 ### `auth`
 
-The `auth` attribute may also contain other data, like `accountName`, `folderPath` or `frequency`. As `auth` should only be used for authentication mechanisms, those two values should disappear soon.
+- `login`: {string} Main credentials field, displayed as default accountName. Some konnectors do not use a `login` parameter, but `identifier` or `email`. In this case the attribute role: 'identifier' should be added to parameter object in manifest.
+- `accountName`: {string} If defined will be displayed preferentially than login or email or parameter with `role: 'identifier'`.
```

Comment:
Where will it be displayed? Should we use this field to generate the konnector's folder path when `defaultFolderPath` is not defined?